src/components/Content.js

```diff
@@ -1,19 +1,90 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from 'react';
+import PropTypes from 'prop-types';
+import './all.sass';
 
-export const HTMLContent = ({ content, className }) => (
-  <div className={className} dangerouslySetInnerHTML={{ __html: content }} />
-)
+export const HTMLContent = ({ content, className }) => {
```

Comment:
What is all this for?

Reply:
It's for rendering the imageblock in the markdown. It finds the object and creates a div with text and image